launcher/launcherdirs.h

```diff
@@ -13,11 +13,13 @@
 class CLauncherDirs
 {
 public:
-	CLauncherDirs();
+	static void prepare();
+	static const CLauncherDirs & get();
```

Comment:
Another option is to completely remove singleton and return default-constructed copy:
```cpp
static CLauncherDirs get();
```
Class does not have any sort of state so construction cost is zero

Reply:
Or replace class with a namespace.

Reply:
even better indeed